Update Set-CsOnlineVoicemailUserSettings.md (#7804)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md

## SYNTAX

```
New-CsOnlineVoicemailPolicy -Identity <XdsIdentity> [-Tenant <Guid>] [-EnableTranscription <Boolean>] [-ShareData <String>]
 [-EnableTranscriptionProfanityMasking <Boolean>] [-EnableTranscriptionTranslation <Boolean>] [-EnableEditingCallAnswerRulesSetting <Boolean>] [-MaximumRecordingLength <Duration>] [-PrimarySystemPromptLanguage <String>] [-SecondarySystemPromptLanguage <String>] [-Force] [-WhatIf] [-Confirm]
 [<CommonParameters>]
```

## DESCRIPTION
Online Voicemail service provides organizations with voicemail deposit capabilities for Phone System implementation.

By default, users enabled for Phone System will be enabled for Online Voicemail. The Online Voicemail policy controls whether or not voicemail transcription, profanity masking for the voicemail transcriptions, translation for the voicemail transcriptions, and editing call answer rule settings are enabled for a user. The policies also specify the voicemail maximum recording length for a user and the primary and secondary voicemail system prompt languages. 

> - Online Voicemail transcription is enabled by default
> - Transcription profanity masking is disabled by default
> - Transcription translation is enabled by default
> - Editing call answer rule settings is enabled by default
> - Voicemail maximum recording length is set to 5 minutes by default
> - Primary and secondary system prompt languages are set to null by default and the users voicemail language setting is used
>  
> Tenant admin would be able to create customized online voicemail policy to match the organization's requirements.

Reviewer: ```suggestion
- Online Voicemail transcription is enabled by default
- Transcription profanity masking is disabled by default
- Transcription translation is enabled by default
- Editing call answer rule settings is enabled by default
- Voicemail maximum recording length is set to 5 minutes by default
- Primary and secondary system prompt languages are set to null by default and the user's voicemail language setting is used
 
Tenant admin would be able to create a customized online voicemail policy to match the organization's requirements.
```

Accept wildcard characters: False
```

### -PrimarySystemPromptLanguage

> The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.

Reviewer: ```suggestion
The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

Reviewer: Links to articles on docs.microsoft.com must use relative links (/folder/topic-file-name) and not absolute links (https://docs.microsoft.com/folder/topic-file-name).

```yaml
Type: String
Parameter Sets: (All)
Aliases: 
Applicable: Skype for Business Online

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -SecondarySystemPromptLanguage

> The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.

Reviewer: ```suggestion
The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```
